Dockerfile

```diff
@@ -1,117 +1,155 @@
 FROM ubuntu:18.04 as build-dep
 
-# Use bash for the shell
-SHELL ["bash", "-c"]
+# Use bash for the SHELL
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN echo "Etc/UTC" > /etc/localtime && \
+    apt-get update && apt-get --no-install-recommends -y install \
+    apt-utils \
+    wget \
+    python \
+    ca-certificates
 
 # Install Node v12 (LTS)
 ENV NODE_VER="12.16.1"
-RUN	ARCH= && \
+RUN ARCH= && \
     dpkgArch="$(dpkg --print-architecture)" && \
-  case "${dpkgArch##*-}" in \
-    amd64) ARCH='x64';; \
-    ppc64el) ARCH='ppc64le';; \
-    s390x) ARCH='s390x';; \
-    arm64) ARCH='arm64';; \
-    armhf) ARCH='armv7l';; \
-    i386) ARCH='x86';; \
-    *) echo "unsupported architecture"; exit 1 ;; \
-  esac && \
-    echo "Etc/UTC" > /etc/localtime && \
-	apt update && \
-	apt -y install wget python && \
-	cd ~ && \
-	wget https://nodejs.org/download/release/v$NODE_VER/node-v$NODE_VER-linux-$ARCH.tar.gz && \
-	tar xf node-v$NODE_VER-linux-$ARCH.tar.gz && \
-	rm node-v$NODE_VER-linux-$ARCH.tar.gz && \
-	mv node-v$NODE_VER-linux-$ARCH /opt/node
+    case "${dpkgArch##*-}" in \
+        amd64) ARCH='x64';; \
+        ppc64el) ARCH='ppc64le';; \
+        s390x) ARCH='s390x';; \
+        arm64) ARCH='arm64';; \
+        armhf) ARCH='armv7l';; \
+        i386) ARCH='x86';; \
+        *) echo "unsupported architecture"; exit 1 ;; \
+    esac && \
+    cd ~ && \
+    wget https://nodejs.org/download/release/v${NODE_VER}/node-v${NODE_VER}-linux-${ARCH}.tar.gz && \
+    tar xf node-v${NODE_VER}-linux-${ARCH}.tar.gz && \
+    rm node-v${NODE_VER}-linux-${ARCH}.tar.gz && \
+    mv node-v${NODE_VER}-linux-${ARCH} /opt/node
 
 # Install jemalloc
 ENV JE_VER="5.2.1"
-RUN apt update && \
-	apt -y install make autoconf gcc g++ && \
-	cd ~ && \
-	wget https://github.com/jemalloc/jemalloc/archive/$JE_VER.tar.gz && \
-	tar xf $JE_VER.tar.gz && \
-	cd jemalloc-$JE_VER && \
-	./autogen.sh && \
-	./configure --prefix=/opt/jemalloc && \
-	make -j$(nproc) > /dev/null && \
-	make install_bin install_include install_lib
-
-# Install ruby
-ENV RUBY_VER="2.6.5"
+RUN apt-get update && apt-get --no-install-recommends -y install make autoconf gcc g++ && \
+    cd ~ && \
```

Comment:
is ``~`` ``/root/`` or ``/home/mastodon``?

Reply:
As I mentioned, this code review is just for readability not functionality.
As this is a container, I believe it'll be /root/ but we can test.